### ISSUE TITLE
fetch one file in the fetchexclude dir

### ISSUE THREAD

**Issue by DJJo14** [git-lfs/git-lfs#5360]
One option I can think of is to set your global git config to exclude all test files in a specific directory:
```
[lfs]
        fetchexclude = large-tests/*
```
You could separate your test data by size into different directories, then developers could pick and choose which file(s)/directories to pull.

`git lfs pull --include "large-tests/*`
or
`git lfs pull --include "large-tests/test.dat"`

_Originally posted by @pmartindev in https://github.com/git-lfs/git-lfs/issues/5351#issuecomment-1523337466_

**Comment by bk2204** [comment]
No, I don't think this is a bug.  If you don't specify exclude files on the command line, they're read from the file, which is the documented behaviour in the `git-lfs-fetch(1)` manual page.
